src/components/Button.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import PropTypes from "prop-types";
+export default class Button extends React.Component {
```

Comment:
Only `App` should be a `class` component. Please make sure this is a functional component